src/libraries/System.Security.Cryptography/ref/System.Security.Cryptography.cs

```diff
@@ -2908,6 +2908,7 @@ public sealed partial class X509ChainPolicy
         public System.TimeSpan UrlRetrievalTimeout { get { throw null; } set { } }
         public System.Security.Cryptography.X509Certificates.X509VerificationFlags VerificationFlags { get { throw null; } set { } }
         public System.DateTime VerificationTime { get { throw null; } set { } }
+        public System.Security.Cryptography.X509Certificates.X509ChainPolicy Duplicate() { throw null; }
```

Comment:
Why not use `Clone`? is this some specific convention?

Reply:
Clone would be fine. I don't care that much about the name. The big question is is if we want to add public method here (like I did) or if we hide it and make it visible only to runtime.

Reply:
I thought that since we have `MemberwiseClone` then it is more common to use `Clone` instead of `Duplicate` or other names, but I don't have too strong opinion on this.